hyrax.gemspec

```diff
@@ -38,6 +38,7 @@ EOF
   spec.add_dependency 'browse-everything', '>= 0.10.5'
   spec.add_dependency 'blacklight', '~> 6.9'
   spec.add_dependency 'blacklight-gallery', '~> 0.7'
+  spec.add_dependency 'devise', '<= 4.4.99'
```

Comment:
Maybe drop in a comment here about why this is locked?

Reply:
is the commit message good enough?

Reply:
Not really, because it would be several steps:
1. Look at the gemspec and say, "Hmm, why is this here"
2. then `git blame` and figure out it was this commit. Which says "See #3210"
3 Open up github and find issue 3210 

It would be kinder to future code readers to just drop a comment here.